spiped/main.c

```diff
@@ -71,7 +71,8 @@ main(int argc, char * argv[])
 	int opt_F = 0;
 	int opt_j = 0;
 	const char * opt_k = NULL;
-	intmax_t opt_n = 0;
+	int opt_n_set = 0;
+	uintmax_t opt_n = 0;
```

Comment:
if we still have `intmax_t opt_n`, then clang complains with:

```
main.c:149:13: error: implicit conversion changes signedness: 'unsigned long' to
      'intmax_t' (aka 'long') [-Werror,-Wsign-conversion]
                                opt_n = SIZE_MAX;
                                      ~ ^~~~~~~~
```
(amongst other similar complaints)